tools/Build/Build.cs

```diff
@@ -50,5 +68,38 @@ void CodeGen(bool verify)
 
 			RunDotNet(toolPath, "conformance/ConformanceApi.fsd", "conformance/", "--newline", "lf", verifyOption);
 		}
+
+		void CreatePipPackage()
+		{
+			CopyFiles("src", "pip/facilitypython", "*.py");
+			AppRunner.RunApp("python", "-m pip install --user --upgrade setuptools wheel twine".Split());
```

Comment:
```suggestion
			RunApp("python", "-m pip install --user --upgrade setuptools wheel twine".Split());
```

Add `using static Faithlife.Build.DotNetRunner;` up top.

Reply:
Maybe that's the wrong `using` statement. I get `The name 'RunApp' does not exist in the current context`

Reply:
Sorry, I meant `using static Faithlife.Build.AppRunner;`

Reply:
Ah, it's `using static Faithlife.Build.AppRunner;`